docs/src/pages/api-reference/react.mdx

```diff
@@ -67,15 +67,17 @@ export const OurUploadDropzone = () => (
-| input                  | string   | If set on the fileroute | Added in `v5.0` | Object matching the input schema that was set for this endpoint in your File Router |
-| onClientUploadComplete | function | No                      |                 | `{fileKey: string, fileUrl: string}[]` callback function when upload is completed   |
-| onUploadError          | function | No                      |                 | callback function when upload fails                                                 |
-| onUploadProgress       | function | No                      | Added in `v5.1` | callback function for upload progress                                               |
-| onUploadBegin          | function | No                      | Added in `v5.4` | callback function for upload begin                                                  |
+| Prop                   | Type               | Required                | Notes           | Description                                                                             |
+| :--------------------- | :----------------- | :---------------------- | :-------------- | :-------------------------------------------------------------------------------------- |
+| `<FileRouter>`         | generic            | Yes                     |                 | The type from the FileRouter you defined in your backend                                |
+| endpoint               | string             | Yes                     |                 | The name of the [route](./server#FileRouter) you want this button to upload to          |
+| input                  | string             | If set on the fileroute | Added in `v5.0` | Object matching the input schema that was set for this endpoint in your File Router     |
+| onClientUploadComplete | function           | No                      |                 | `{fileKey: string, fileUrl: string}[]` callback function when upload is completed       |
+| onUploadError          | function           | No                      |                 | callback function when upload fails                                                     |
+| onUploadProgress       | function           | No                      | Added in `v5.1` | callback function for upload progress                                                   |
+| onUploadBegin          | function           | No                      | Added in `v5.4` | callback function for upload begin                                                      |
+| config                 | object             | No                      | Added in `v5.5` | object to pass additional configuration for dropzone                                    |
+| config.mode            | 'auto' \| 'manual' | No                      | Added in `v5.5` | mode of dropzone. 'manual' is default one. 'auto' triggers upload right after selection |
```

Comment:
v5.4 isn't out yet, so we should verify what version this will land on before merging